### PR TITLE
Bump/v7.0.0

### PR DESCRIPTION
**Pull request by JoeChapman**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "hof-bootstrap",
   "description": "A bootstrap for HOF projects",
-  "version": "6.0.2",
+  "version": "7.0.0",
```
Comment (joefitter) [inline package.json]: can we just PR the changelog and npm version major _after_ this is merged

Comment (JoeChapman) [inline package.json]: do you mean two separate PR's, one for the change log and one for the version number? Why?

Comment (joefitter) [inline package.json]: Because this is _only_ updating the version number in package.json, meaning package.json isn't in sync with npm or github. The person who merges the PR for changelog should update master and run `npm version major; git push origin master --tags; npm publish`

Comment (joefitter) [inline package.json]: we have done it this way for a while

**Comment by JoeChapman** [comment]
@joefitter updated
